### ISSUE TITLE
Perennial deciduous fruit trees in CLM

### ISSUE THREAD

**Issue by odombro** [ESCOMP/CTSM#1956]
I've been working on a crop development to consider perennial deciduous fruit trees in CLM5 to model fruit orchards ([the published paper](https://gmd.copernicus.org/articles/15/5167/2022/)).
The development includes a new phenology subroutine for perennial deciduous woody crops and considers the typical phenological development, carbon allocation and management practices for fruit trees grown in commercial orchards. I've also parameterized a new crop PFT for apple trees.  
This issue relates to #1908 and I've been in touch with Ashehad and Yuanchao in the early stages of my development. I've used some of their features in a similar way (e.g. a perennial flag, annual harvest implementation, orchard rotation).
Thus far the model was developed, calibrated and tested on the point scale.

Best,
Olga

**Comment by wwieder** [comment]
Excellent, thanks for opening up this issue @odombro.  Do you plan to open up a PR for your fruit tree work?  If so, maybe @danicalombardozzi or @samsrabin can help get this ready?  As a note, with other developments we've brought in PR's that are functional, but not on by default (e.g. MIMICS).  I wonder if this is something we want to consider here so that others can contribute to this code down the road?

**Comment by odombro** [comment]
Yes, I'm happy to get started on a PR for the fruit tree development. I'll get in touch with @danicalombardozzi and @samsrabin

**Comment by wwieder** [comment]
This is still an exciting feature, but one that won't make it into the CLM6 release.
